_data/legal.yml

```
@@ -21,6 +21,7 @@ websites:
       img: hellosign.png
       tfa: Yes
       sms: Yes
+      software: Yes
       exceptions:
-          text: "Two-factor authentication is only available to paid Business plan senders and above."
-      doc: https://faq.hellosign.com/hc/en-us/articles/205714548-How-do-I-set-up-two-factor-authentication-
+          text: "SMS Two-factor authentication is only available to paid Business plan senders and above."
+      doc: https://faq.hellosign.com/hc/en-us/articles/360025164091-Two-Factor-Authentication-Google-Authenticator
```

On Zendesk pages, the text that's behind the numbers in the URL aren't actually needed.

```suggestion
      doc: https://faq.hellosign.com/hc/en-us/articles/360025164091
```

---

Good suggestion, thanks!